### PR TITLE
Collapse Published Date row when date is null

### LINKED ISSUE #34: Published date from metadata not shown
Seems like the publishing date from the package metadata is always missing?

image

### PR DESCRIPTION
**Pull request by ivangrek**
fixes #34

### CODE REVIEW

**Review by mast-eu — APPROVED** [review]
Works as intended.

File: src/PackageManager/ViewModels/PackageViewModel.cs [hunk 1]
```diff
@@ -25,6 +24,10 @@ public class PackageViewModel : ObservableModel
         public string Description => model.Description;
         public string Authors => model.Authors;
         public DateTime? Published => model.Published;
+        public Visibility PublishedVisibility => Published is null
```
Comment (maraf) [inline src/PackageManager/ViewModels/PackageViewModel.cs:27]: I think this should be handled by value converter in xaml.

Comment (ivangrek) [reply src/PackageManager/ViewModels/PackageViewModel.cs:27]: Is there a fundamental difference between a converter and a view model property?

Comment (ivangrek) [reply src/PackageManager/ViewModels/PackageViewModel.cs:27]: As I understand Published is not PublishedVisibility.
Projection into different context.

Comment (maraf) [reply src/PackageManager/ViewModels/PackageViewModel.cs:27]: `PublishedVisibility` doesn't any value in the viewmodel's point of view. It's only for presentation purposes, which should be handled by xaml in mvvm.

Comment (ivangrek) [reply src/PackageManager/ViewModels/PackageViewModel.cs:27]: Is PackageViewModel "M" not "VM" in this case? I looked at the suffix.

Comment (maraf) [reply src/PackageManager/ViewModels/PackageViewModel.cs:27]: It is a VM, but for exactly these use-cases we have a [NullToCollapsedConverter registered in App.xaml](https://github.com/gitextensions/gitextensions.pluginmanager/blob/master/src/PackageManager.UI/App.xaml#L29).

**Pull request merged**